### PR TITLE
Added framework requirements and test objectives

### PR DESCRIPTION
**Pull request by Bookanakere**
Hi Pramod, 

Instead of updating the existing readme file, I created two new files one for requirements and the other one for test objectives. Could you please take a look?

Regards,
Rashmi

### CODE REVIEW

**Comment by PramodKumarYadav** [comment]
> Hi Pramod,
> 
> Instead of updating the existing readme file, I created two new files one for requirements and the other one for test objectives. Could you please take a look?
> 
> Regards,
> Rashmi

So in general, you only have one readme.md file Rashmi. Idea is any new team member can get everything they need from one readme.md file. You can give a link of one file to another but generically its one readme.md file per project. So maybe you can merge changes and put them into one file.

**Comment by PramodKumarYadav** [comment]
updating readme file in this PR.

**Comment by Bookanakere** [comment]
Hi Pramod, I have resolved the conflicts and pushed the updated branch to remote. Please take a look and approve the PR.

PS: Turns out, I had to stage the file again before committing. Thats why I was facing the issue with commit.

**Comment by PramodKumarYadav** [comment]
> Hi Pramod, I have resolved the conflicts and pushed the updated branch to remote. Please take a look and approve the PR.
> 
> PS: Turns out, I had to stage the file again before committing. Thats why I was facing the issue with commit.

Nice. If I would have seen two files (one in staging and one with changes, it was easy to track but lack of visibility made it difficult to see what was going on). Maybe lets do zoom from tomorrow if it works better?

File: README.md [hunk 1]
```diff
@@ -25,6 +25,99 @@ Powershell
...
+- [] Choice of browser is to be made outside tests (either via CI or command line). In case if you dont want to do that, there should be a option for user to define a default driver with which all tests could be run.
+- [] Tests should be atomic and independent of each other (to allow running in parallel)
+- [] Tests should run in parallel (to keep test feedback fast and to encourage more tests)
+- [] se of docker files to setup test environment
+- [] To allow tests to run distributed on a grid (not same as parrallel)
+- [] To allow a consistent setup across test machines.
+- [] To create a test container to also run tests from container (entrypoint) -not just from localhost -GRID (which needs local setup)
+- [] Use of CI to run tests with each merge in master
+- [] Use of reports (html) & CI parceable
 # Files that matters:
```
Comment (PramodKumarYadav) [inline README.md:104]: - [x] this part of line is duplicate from above (and maybe also next 2/3 lines). This should also let you get some more practise on what needs to be done in this case. We can discuss more tomorrow.

**Review by PramodKumarYadav — CHANGES_REQUESTED** [review]

**Review by PramodKumarYadav — APPROVED** [review]

**Pull request merged**